domain/model/modelmigration/import_test.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package modelmigration
+
+import (
+	"context"
+
+	"github.com/juju/description/v5"
+	"github.com/juju/errors"
+	"github.com/juju/names/v5"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	coremodel "github.com/juju/juju/core/model"
+	modeltesting "github.com/juju/juju/core/model/testing"
+	coreuser "github.com/juju/juju/core/user"
+	"github.com/juju/juju/domain/credential"
+	"github.com/juju/juju/domain/model"
+	usererrors "github.com/juju/juju/domain/user/errors"
+	"github.com/juju/juju/environs/config"
+	jujuversion "github.com/juju/juju/version"
+)
+
+type TestImportService struct {
```

Comment:
Same again, use gomock for consistency with the other tests. It's just one EXEPCT() call.